chat-plugins/trivia.js

```diff
@@ -353,14 +353,14 @@ class Trivia extends Rooms.RoomGame {
 	}
 	// Formats the player list for display when using /trivia players.
 	formatPlayerList() {
-		return Object.keys(this.players)
-			.map(userid => {
-				let player = this.players[userid];
-				let username = player.name;
-				if (player.isAbsent) return `<span style="color: #444444">${username}</span>`;
-				return username;
-			})
-			.join(', ');
+		let playerList = Object.values(this.players);
+		playerList.sort(function (p1, p2) {
```

Comment:
This shold be an arrow function. I think the formatting's a bit verbose for what the code's doing:
return Object.values(this.players)
    .sort((p1, p2) => p2.score - p1.score)
    .map(...)
    .join(',');
```